bare-metal/container-linux/kubernetes/fl/controller.yaml.tmpl

```diff
@@ -0,0 +1,171 @@
+---
```

Comment:
I'd like to keep the Ignition for Container Linux derivatives unified. If I allowed Flatcar to drift, I might have to allow future derivatives and I'd like to keep everything aligned for a while